src/tests/test_dte_data_models.py

```diff
@@ -1579,24 +1722,27 @@ def test_validate_referencias_rut_otro_is_consistent_with_tipo_dte(self) -> None
 
         expected_validation_errors = [
             {
-                'loc': ('__root__',),
-                'msg': "Setting a 'rut_otro' is not a valid option for this 'tipo_dte':"
-                " 'tipo_dte' == <TipoDte.FACTURA_ELECTRONICA: 33>,"
-                " 'Referencia' number 1.",
+                'loc': (),
+                'msg': "Value error, Setting a 'rut_otro' is not a valid option for this "
+                "'tipo_dte': 'tipo_dte' == <TipoDte.FACTURA_ELECTRONICA: 33>, "
+                "'Referencia' number 1.",
                 'type': 'value_error',
             },
         ]
 
-        with self.assertRaises(pydantic.v1.ValidationError) as assert_raises_cm:
+        with self.assertRaises(pydantic.ValidationError) as assert_raises_cm:
             dataclasses.replace(
                 obj,
                 referencias=[obj_referencia],
             )
 
-        validation_errors = assert_raises_cm.exception.errors()
+        validation_errors = assert_raises_cm.exception.errors(
+            include_url=False,
+            include_context=False,
+            include_input=False,
```

Comment:
Sort lexicographically.

```suggestion
            include_context=False,
            include_input=False,
            include_url=False,
```